pkg/bindinfo/handle.go

```diff
@@ -71,9 +69,12 @@ type BindHandle struct {
 	bindInfo struct {
 		sync.Mutex
 		atomic.Value
-		lastUpdateTime types.Time
 	}
 
+	// lastUpdateTime records the last update time for the global sql bind cache.
+	// This value is used to avoid reload duplicated bindings from storage.
+	lastUpdateTime types.Time
```

Comment:
Can I ask why do we need to move it out?

Reply:
To make the global cache clearer, since actually this variable is only used to record the last `Update` time, which should not be put in the global cache structure.